app/scripts/lib/app-start.js

```diff
@@ -173,6 +174,37 @@ function (
       };
     },
 
+    _checkForRefresh: function (currentPath) {
+
+      var storage = Storage.factory('sessionStorage', this._window);
+
+      var refreshMetrics = storage.get('last_page_loaded');
+
+      if (refreshMetrics && refreshMetrics.url === currentPath) {
+        if (this._metrics) {
+          this._metrics.logEvent('refresh(' + currentPath + ')');
```

Comment:
Yes, although you'd need access to the view in order to use those. It seems like this check might fit better in the router, in which case you'd have access to the view.